src/Application.php

```diff
@@ -230,48 +223,6 @@ public function getEventManager()
         return $this->events;
     }
 
-    /**
-     * Static method for quick and easy initialization of the Application.
-     *
-     * If you use this init() method, you cannot specify a service with the
-     * name of 'ApplicationConfig' in your service manager config. This name is
-     * reserved to hold the array from application.config.php.
-     *
-     * The following services can only be overridden from application.config.php:
-     *
-     * - ModuleManager
-     * - SharedEventManager
-     * - EventManager & Laminas\EventManager\EventManagerInterface
-     *
-     * All other services are configured after module loading, thus can be
-     * overridden by modules.
-     *
-     * @param array $configuration
-     * @return Application
-     */
-    public static function init($configuration = [])
-    {
-        // Prepare the service manager
-        $smConfig = isset($configuration['service_manager']) ? $configuration['service_manager'] : [];
-        $smConfig = new Service\ServiceManagerConfig($smConfig);
-
-        $serviceManager = new ServiceManager();
-        $smConfig->configureServiceManager($serviceManager);
-        $serviceManager->setService('ApplicationConfig', $configuration);
-
-        // Load modules
-        $serviceManager->get('ModuleManager')->loadModules();
-
-        // Prepare list of listeners to bootstrap
-        $listenersFromAppConfig     = isset($configuration['listeners']) ? $configuration['listeners'] : [];
-        $config                     = $serviceManager->get('config');
-        $listenersFromConfigService = isset($config['listeners']) ? $config['listeners'] : [];
-
-        $listeners = array_unique(array_merge($listenersFromConfigService, $listenersFromAppConfig));
-
-        return $serviceManager->get('Application')->bootstrap($listeners);
-    }
-
```

Comment:
Why are you removing this method?

Removal of this MUST be called out in both the CHANGELOG and the documentation, as our own skeleton has consistently used this method in order to initialize and bootstrap the application from aggregated configuration. If it goes away, users need to know what they need to change in their `public/index.php`.

Reply:
This method no longer provides utility it once did.
With the the module manager dropped, responsibility to setup the application pushed completely out of the scope of this package.

It is now entirely up to skeleton application to provide the default wiring. As initially outlined in the [Zend\Mvc v4 RFC](https://discourse.laminas.dev/t/rfc-zend-mvc-4-design-changes/447) the goal is to move towards current mezzio skeleton application setup. Config+container proved to be flexible enough.

At this time I am working on a new feature that would somewhat approximate laminas-httphanderrunner and would affect how `public/index.php` needs to be changed.

On a side note, I have no immediate intention to provide update for the laminas-modulemanager. I can easily see it retired unless there is a demand for the different set of features not available with the config aggregator.